### DISCUSSION TITLE
SyntaxError: Unexpected token 'export'

### DISCUSSION THREAD

**Discussion by omgui** [remarkjs/remark#987] [General] [answered]
When using react-markdown in the latest version of create-react-app I can no longer run jest due to the following error:

<img width="404" alt="image" src="https://user-images.githubusercontent.com/92113175/165271759-7a68d9b2-dd92-428c-a8eb-4df128594087.png">

It all works fine when I remove react-markdown. I have tried several examples of adding --transformIgnorePattern but this does not seem to fix the problem.

**Comment by wooorm** [comment, marked as answer]
[This is a Jest + ESM problem. It has been asked multiple times in different places.](https://github.com/orgs/remarkjs/discussions?discussions_q=jest+esm)
